test/e2e/framework/network_pods.go

```diff
@@ -177,6 +177,7 @@ func (np *NetworkPod) AwaitFinishVerbose(verbose bool) {
 			fmt.Printf("%s", removeDupDataplaneLines(np.TerminationMessage))
 		}
 	}
+	return np.TerminationMessage
```

Comment:
What is the reason for explicitly returning `np.TerminationMessage` when callers can just access it themselves?